docs/js/doc-links.yml

```diff
@@ -1,263 +1,263 @@
 - title: First Steps
   collapsable: false
-  link: /docs/pre-release/
+  link: /docs/latest/
```

Comment:
@khussey Is this right? You didn't object to it in the checklist, but now I'm a little confused that it's showing up here--I though `release/v1.5` should have already had it saying "latest"?

Reply:
Yes, it should already have said "latest" (and currently does on the website); where did this version of docs-links.yml come from?

Reply:
Ah, it looks like the `rel/v1.5.3` branch is based on c13293f (the 1.5.2 tag?), which was before the URLs were corrected to "latest" (f9ee618) on `release/v1.5`.